quinn-proto/src/packet.rs

```diff
@@ -90,7 +90,7 @@ impl PartialDecode {
     }
 
     /// Length of QUIC packet being decoded
-    pub(crate) fn len(&self) -> usize {
+    pub fn len(&self) -> usize {
```

Comment:
this is only made public for the placeholder assertion in the test and not necessarily required.

Reply:
That's fine since this type isn't reexported under normal conditions anyway.

Reply:
Though, on review, this isn't (at least currently) a very interesting property to check.